spec/controllers/organizations_controller_spec.rb

```diff
@@ -34,6 +35,22 @@
       end
     end
 
+    context 'user not in the classroom' do
+      before do
+        sign_in(student)
+      end
+
+      it 'will be added if he is admin of the GitHub organization' do
+        get :index
+        expect(assigns(:current_user).organizations.first.id).to eq(organization.id)
+      end
+
+      it 'will not be added if he is not admin of the GitHub organization' do
+        get :index
+        expect(assigns(:current_user).organizations.length).to eq(0)
+      end
+    end
+
```

Comment:
@dinever this is a good start, what do you think about something like this?

```ruby
    context 'user with admin privilege on the organization but not apart of the classroom' do
      before(:each) do
        organization.users = []
      end

      it 'will add the user to the classroom' do
        expect(user.organizations).to be_empty

        get :index

        expect(user.organizations).not_to be_empty
        expect(user.organizations).to include(organization)
      end
    end

    context 'user without admin privilege on the organization' do
      before(:each) do
        sign_out
        sign_in(student)
      end

      it 'will not add the user to the classroom' do
        expect(student.organizations).to be_empty

        get :index

        expect(student.organizations).to be_empty
      end
    end
```

/cc @johndbritton for feedback